### DISCUSSION TITLE
Webgl replacement canvas

### DISCUSSION THREAD

**Discussion by hehos** [ONLYOFFICE/DocumentServer#1893] [Q&A]
Hello, please ask me a question about WebGL replacement Canvas2D scheme

Webgl can get hardware acceleration. If the SDKJS part of the project and the underlying canvas are implemented using WebGL, will the rendering speed and performance be much better?

**Comment by ShockwaveNN** [comment]
Hi, thanks for a question

I've just had a talk with our lead developer

And seems that right now we got no plans to use WebGL explicitly
Some of our code (like text (textures)) already had acceleration
Some feature can be improved, like presentation transition

But even if we fully support WebGL - speed improvements will not be very significant
